tests/v2/test_auth.py

```diff
@@ -13,6 +13,14 @@ def test_create_user(self):
         response_content =  json.loads(response.data.decode())
         self.assertTrue(response_content['status'] == 201)
 
+    def test_user_login(self):
+        """Test that endpoint can login a registered user"""
+        super().create_user(admin_user)
+        response = super().login_user(admin_user_login)
+        self.assertEqual(response.status_code, 200)
+        response_content =  json.loads(response.data.decode())
+        self.assertTrue(response_content['status'] == 200)
+
```

Comment:
Hi, Kindly add more tests for when a user tries to login with the wrong password, without an email, or without a password and when an unexisting user tries to log in

Reply:
Right away!